pyproject.toml

```diff
@@ -24,31 +25,27 @@ keywords=["Reinforcement Learning", "Finance"]
 github = "https://github.com/finrl/finrl-library"
 
 [tool.poetry.dependencies]
-python = ">=3.7"
-pyfolio = {git="https://github.com/quantopian/pyfolio.git#egg=pyfolio-0.9.2"}
+python = "~3.10"
 elegantrl = {git="https://github.com/AI4Finance-Foundation/ElegantRL.git#egg=elegantrl"}
-alpaca_trade_api = ">=2.1.0"
-ccxt = ">=1.66.32"
-exchange_calendars = "3.6.3"
-gputil = "*"
-gym = ">=0.17"
-importlib-metadata = "4.13.0"
-jqdatasdk = "*"
-lz4 = "*"
-matplotlib = "*"
-pandas = ">=1.1.5"
-numpy = ">=1.17.3"
-tensorboardX = "*"
-yfinance = "*"
-stockstats = ">=0.4.0"
-scikit-learn = ">=0.21.0"
-ray = {extras=["default", "tune"], version=">=2.0.0"}
-stable-baselines3 = "^1.6.2"
-wrds = ">=3.1.6"
+alpaca-trade-api = "^3"
+ccxt = "^3"
+exchange-calendars = "^4"
```

Comment:
I think this cause a regression fixed by https://github.com/AI4Finance-Foundation/FinRL/pull/695